ipa_check_consistency

```diff
@@ -609,6 +609,38 @@ dns_zones() {
   fi
 }
 
+certificates() {
+  local server="" count="" state=""
+  local dir="${TMP_DIR}/certificates"
+
+  mkdir -p "$dir"
+  printf "%-${FIRST_COLUMN}s" "Certificates"
+  for server in "${SERVERS[@]}"; do
+    (
+    if ! count=$(query_ldap "ou=certificateRepository,ou=ca,o=ipaca" \
+      "(certStatus=*)" \
+      "dn" "one"); then
+      count="ERROR"
+    else
+      count=$(grep -c "^dn:" <<<"$count" || true)
+    fi
+    printf "%s" "$count" > "${dir}/${server}"
+    ) &
+  done
+  wait
+  for server in "${SERVERS[@]}"; do
+    printf "%-${MIDDLE_COLUMNS}s" "$(< "${dir}/${server}")"
+  done
+  state="$(is_consistent "$dir")"
+  printf "%-${LAST_COLUMN}s\n" "$state"
+  if [[ $NAGIOS -eq 1 ]]; then
+    printf "%s\n" "$state" >> "${TMP_DIR}/nagios"
+    if [[ "$NAGIOSREPORT" == "zones" ]]; then
```

Comment:
Also, this change is only valid for CA-full installation. In my CA-less deployment I get ERRORs for all my nodes:

`Certificates        ERROR      ERROR      ERROR      ERROR      ERROR      ERROR      OK`

IMO, it would make sense to check if CA is available (i.e. by checking if LDAP path exists) and include the Certificates bit only in CA-full installation.